laravel-mfa-example/resources/views/list_factors.blade.php

```diff
@@ -2,57 +2,67 @@
+                <p>Start selling to enterprise customers with just a few lines of code.</p>
+                <p>Implement features like single sign-on in minutes instead of months.</p>
+            </div>
+            <div class="flex success-buttons">
+                <a href="https://workos.com/signup" target="_blank"><button class='button'>Get Started</button></a>
+                <a href="mailto:sales@workos.com?subject=WorkOS Sales Inquiry" target="_blank"><button
+                        class='button button-outline sales-button'>Contact
+                        Sales</button></a>
+            </div>
+        </div>
   <div class="logged_in_div_right">
   <div class="flex_column">
-                <div class="flex space_between width_40vw">
-                    <div>
-                        <h2>Current MFA Factors</h2>
-                    </div>
+        <div class="flex space_between width_40vw">
+ 
 
-                    <div class="flex space-between">
-                        <a href="/enroll_factor_details">
-                            <button class="button button-outline button-sm">Enroll New Factor</button>
-                        </a>
-                        <a href="/clear_session">
-                            <button class="button button-outline button-sm">Clear Factors</button>
-                        </a>
-                    </div>
-                </div>
+            <div class="flex space-between">
+                <a href="/enroll_factor_details">
+                    <button class="button button-outline button-sm">Enroll New Factor</button>
+                </a>
+                <a href="/clear_session">
+                    <button class="button button-outline button-sm">Clear Factors</button>
+                </a>
+            </div>
+        </div>
 
-                <div class="flex-column">
-                @if ($factors)
-                    @foreach($factors as $factor)
-                        <div>
-                            <a class="button login_button" href="/factor_detail?id={{$factor->id}}">{{$factor->id}} | {{$factor->type}}</a>
-                        </div>
-                    @endforeach                 
-                @else
-                <div class="mt-20 card width_40vw">
-                    <p style="text-align: center; color: gray;">No Current Factors</p>
+        <div class="flex-column">
+        <div>
+                <h2>Current MFA Factors</h2>
+        </div>
+        @if ($factors)
+            @foreach($factors as $factor)
+                <div>
+                    <a class="button login_button" href="/factor_detail?id={{$factor->id}}">{{$factor->id}} | {{$factor->type}}</a>
                 </div>
-                @endif
-            </div>
-            </div>
+            @endforeach                 
+        @else
+        <div class="mt-20 card width_40vw">
+            <p style="text-align: center; color: gray;">No Current Factors</p>
```

Comment:
Can this width be wider? I think if you switch the class to width-40vw it will pick up the class, this was changed from the underscore to be more consistent across the apps. 
![CleanShot 2022-08-26 at 14 43 07](https://user-images.githubusercontent.com/47123183/186987983-085e04ef-3fa0-41e4-aa5f-6945367d11b3.png)

Reply:
done!